pcdet/spconv_utils.py

```diff
@@ -0,0 +1,34 @@
+from typing import Set
```

Comment:
Please move this file to ``pcdet/utils/spconv_utils.py`` since it is strange to list this file under ``pcdet/``.